tools/klee/main.cpp

```diff
@@ -1330,8 +1331,7 @@ int main(int argc, char **argv, char **envp) {
   // locale and other data and then calls main.
   Function *mainFn = mainModule->getFunction(EntryPoint);
   if (!mainFn) {
-    llvm::errs() << "'" << EntryPoint << "' function not found in module.\n";
-    return -1;
+    klee_error("'%s' function not found in module.\n", EntryPoint.c_str());
```

Comment:
@MartinNowack as I mentioned before, I would rather stick to `klee_error` here since the original code was exiting anyways.